includes/class-reader-activation.php

```diff
@@ -555,6 +567,10 @@ public static function render_auth_form() {
 		}
 		// phpcs:enable
 
+		if ( $is_inline ) {
+			$classnames[] = 'newspack-reader__auth-form__inline';
```

Comment:
```suggestion
			$classnames[] = $class( 'inline' );
```

Reply:
Ooh, I misread what that function did. Much cleaner.